lang/py/src/avro/protocol.py

```diff
@@ -20,10 +20,12 @@
   from hashlib import md5
 except ImportError:
   from md5 import md5
+
```

Comment:
Usually we'd try to avoid space or blank line only changes to the existing repo in the open source project, especially lots of blank changes, such as changing all the indentation. I think this is fine because it only has a couple of changes, and this is just something to keep in mind. :)